doc/ext/imgconverter.rst

```diff
@@ -16,6 +18,12 @@ Internally, this extension uses Imagemagick_ to convert images.
 
 .. _Imagemagick: https://www.imagemagick.org/script/index.php
 
+.. note:: Imagemagick rasterizes a SVG image on conversion.  As a result, the image
+          becomes not scalable.  To avoid that, please use other image converters
+          like sphinx_svg_to_pdf.py_ (which uses Inkscape).
+
+.. _sphinx_svg_to_pdf.py: https://gist.github.com/Balletie/a3ccf434f8ec492e47fff4589225d5cb
```

Comment:
`sphinxcontrib-svg2pdfconverter` might be better.
https://github.com/missinglinkelectronics/sphinxcontrib-svg2pdfconverter